functions.php

```diff
@@ -25,7 +27,30 @@ function linkDB(): PDO
 
 function getDataFromDB(PDO $db): array
 {
-    $query = $db->query("SELECT `name`, `brand`, `image`, `weight` FROM `computerMice` WHERE (`name` IS NOT NULL AND `brand` IS NOT NULL);");
+    $query = $db->query("SELECT `name`, `brand`, `image`, `weight`, `is_wireless` FROM `computerMice` 
+    WHERE (`name` IS NOT NULL AND `brand` IS NOT NULL AND `weight` IS NOT NULL AND `is_wireless` IS NOT NULL);");
     return $query->fetchAll();
 }
 
+function formatName(string $name): string
+{
+    $name = trim($name, " \t\n\r\0\x0B");
+    $name = ucwords($name);
+    return $name;
+}
+
+function validateInfo(string $mouseName,string $mouseBrand,string $mouseWeight): string
+{
+    if ((strlen($mouseName) > 100) || (strlen($mouseBrand) > 50) || ($mouseWeight > 1000) || ($mouseWeight < 1)) {
+        return 'inputLength';
+    } elseif (!(filter_var($mouseWeight, FILTER_VALIDATE_INT))) {
+        return 'integer';
+    } elseif ((!(preg_match("/^([a-zA-Z0-9- ]+)$/", $mouseName))) || (!(preg_match("/^([a-zA-Z0-9- ]+)$/", $mouseBrand)))) {
+        return 'invalidName';
+    } else {
+        return 'true';
```

Comment:
validate functions like this should just return true or false, is the data valid or isnt it? if you want to validate different bits of data, you need multiple functions. a function should do 1 thing. this function for 4